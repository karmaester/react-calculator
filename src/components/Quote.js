const Quote = () => (
  <div className="quote">
    &quot; Mathematics in not about numbers, equations, computations,
    or algorithms: it is about understanding.&quot;
    <span>William Paul Thurston</span>
  </div>
);

export default Quote;
