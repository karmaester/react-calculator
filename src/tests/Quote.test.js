import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Quote from '../components/Quote';

test('should render the Home Page', () => {
  const component = renderer.create(
    <BrowserRouter>
      <Quote />
    </BrowserRouter>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
