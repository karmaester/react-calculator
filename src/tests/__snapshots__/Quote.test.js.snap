// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Home Page 1`] = `
<div
  className="quote"
>
  " Mathematics in not about numbers, equations, computations, or algorithms: it is about understanding."
  <span>
    William Paul Thurston
  </span>
</div>
`;
